pr_check_docker.sh

```diff
@@ -23,7 +23,7 @@ PGDATA=/var/lib/postgresql/data /usr/lib/postgresql/*/bin/pg_ctl -w stop
 PGDATA=/var/lib/postgresql/data /usr/lib/postgresql/*/bin/pg_ctl start -o "-c listen_addresses='*' -p 5432"
 
 # check the code. Then run the unit and integration tests and cleanup cluster (if running against real OCM)
-make -k lint verify test test/integration test/report-portal/push test/cluster/cleanup 
+make -k lint verify test test/integration/kafka test/report-portal/push test/cluster/cleanup 
```

Comment:
That means we don't run connectors test anymore as part of the CI / release process. Is that intentional?

Reply:
This change is intentional. 

Connectors test are still run as part of the Github CI. 
The only common parts we've with them is that we share the repository (this repo) and the code base so running the connectors integration test in Github CI makes sense. 
The reason for this change is that, the commonality between KFM and COS-Fleet-manager stops at code level, the connector team have a different release process (which is based on https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-manager ) and as far as I am aware, they are not part of the different downstream pipelines dedicated for Managed Kafka. 

/cc @lburgazzoli @dhirajsb to chime in if this should be reverted.

Reply:
While it's true that connectors don't share any build output from kfm downstream, the integration tests can only be run as part of the kfm test process downstream. Since c-f-m pulls k-f-m as a dependency, there's no way to run these integration tests as part of c-f-m downstream CI process. 
Is there an issue in the connector integration tests that's preventing them to be run downstream?

Reply:
@dhirajsb thanks for the comments. I've tagged you in a documentation that outlines the intention behind this: 

In short summary, we are to stop using mas_sso rhoas realm in these pipelines and start using rh sso stage env. Is that possible with the current connectors test?

Reply:
> ..the integration tests can only be run as part of the kfm test process downstream. Since c-f-m pulls k-f-m as a dependency, there's no way to run these integration tests as part of c-f-m downstream CI process.

Would you happen to have a documentation about how this works downstream for the connector team?
"Ideally" and "in my imagination", the way kfm and cfm releases processes should be decouple from one another. 
i.e any failure to kfm code shouldn't impact cfm releases and vice versa. 

One way to run the connectors integration test downstream would still be when the kfm version bumps occurs. The cfm pipeline will then clone the "kfm/common code repo" and run the connectors integration test before doing any releases.

Reply:
Following a discussion in private, I've reverted this change. 

Thanks for calling it out @miguelsorianod @dhirajsb

Reply:
Just to note, I'll still like to progress the discussion around decoupling the the KFM & CFM release process. I've opened a separate discussion topic on that.